pandas/core/frame.py

```diff
@@ -728,11 +728,35 @@ def iteritems(self):
         """
         Iterator over (column name, Series) pairs.
 
+        Iterates over columns as key, value dict-like pairs with columns name as keys and Series as values.
+
+        Returns
+        -------
+        it : generator
+            A generator that iterates over the columns of the frame.
+
         See also
         --------
         iterrows : Iterate over DataFrame rows as (index, Series) pairs.
         itertuples : Iterate over DataFrame rows as namedtuples of the values.
 
+        Examples
+        --------
+        >>> df = pd.DataFrame({'col1': [1, 2], 'col2': [0.1, 0.2]},
+        ...                   index=['a', 'b'])
+        >>> df
+           col1 col2
+        a     1  0.1
+        b     2  0.2
+        >>> for col in df.iteritems():
+        ...     print(col)
```

Comment:
I think something like next would make things easier to understand:
```
for label, content in df.iteritems():
    print('label:', label)
    print('content:' content)
```

Reply:
Thanks for the edits! I rephrased the long description, changed the Yields section to cover label and content, prepended DataFrame. to the See Also examples, and reworked the Examples section to hopefully be more engaging.

Reply:
did you push the changes?

Reply:
I got my git commands mixed up *sheepishly*, but it's pushed now